articles/openshift/howto-using-azure-redhat-openshift.md

```diff
@@ -24,47 +24,12 @@ You'll need the following to create an Azure Red Hat OpenShift 4.3 cluster:
 
 - Azure CLI version 2.0.72 or greater
   
-- The 'az aro' extension
```

Comment:
@lamek `az aro` will not be live in the core CLI next week, instructions about the extension must not be removed for now.